heppi/heppi.py

```diff
@@ -253,60 +253,28 @@ def MakeStatProgression(myHisto,histDwSys={},histUpSys={},
         for ibin in range(myHisto.GetNbinsX()+1):
             y    = statPrecision.GetBinContent(ibin);
             stat = statPrecision.GetBinError  (ibin);
-            
-            
-            valup = {}
-            valdw = {}
-            #vals  = [y]
-            #print "--> Nm[",ibin,"]::  (", y ,")"
-            for sys in histUpSys:
-                val   = histUpSys[sys].GetBinContent(ibin)
-                #vals += [val]
-                valup[sys] = val
-                #print "--> Up[",ibin,"]:: ", sys, " (", val ,")"
-            for sys in histDwSys:
-                val   = histDwSys[sys].GetBinContent(ibin)
-                #vals += [val]
-                valdw[sys] = val
-                #print "--> Dw[",ibin,"]:: ", sys, " (", val ,")"
-                
-            #largest_val  = max(vals)
-            #smallest_val = min(vals)
 
-            systError2   = 0
-            for up in valup:
-                dw = up.replace("Up","Down")
-                e = 0.5*(valup[up] - valdw[dw])
-                #print "--> error[",ibin,"]:: ", up, " (", e ,")"
-                systError2 += e*e 
-                            
-            error  = 0
-            if combine_with_systematic:
-                #math.sqrt(syst*syst + stat*stat) 
-                error = math.sqrt(systError2 + stat*stat)
-            else:
-                #error = 0.5*(largest_val - smallest_val)
-                error = math.sqrt(systError2)
-            #print "--> stat error[",ibin,"]:: (", stat ,")"
-            #print "--> syst error[",ibin,"]:: (", math.sqrt(systError2) ,")"
-            #print "--> combined error[",ibin,"]:: (", error ,")"
-            if (y>0):
-                statPrecision.SetBinContent(ibin,   1)#0.5*(largest_val + smallest_val)/y);
-                statPrecision.SetBinError  (ibin,   error/y );
+            up_errsum2   = 0
+            down_errsum2 = 0
+            if( y > 0 ):
+		up_errsum2   = (stat/y)*(stat/y) 
```

Comment:
The indentation change at this point doesn't look right to me.  @edjtscott, at some point can you create a new area with this PR and confirm it works out of the box?